packages/@sanity/language-filter/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../../tsconfig",
+  "include": ["src"],
+  "compilerOptions": {
+    "composite": true,
+    // todo: we need to disable noImplicitAny in a transition phase and should be removed eventually
+    "noImplicitAny": false,
```

Comment:
since we're converting this package to typescript now, would it be much extra work to not include this? (it's a flag we've traditionally only been setting on older packages that would be too much work to get to strict mode)

Reply:
Can do, yes.